packages/edit-site/src/components/editor/index.js

```diff
@@ -182,10 +182,9 @@ export default function Editor( { isLoading } ) {
 	);
 
 	const settings = useSpecificEditorSettings();
-	const isReady =
-		! isLoading &&
-		( ( postWithTemplate && !! contextPost && !! editedPost ) ||
-			( ! postWithTemplate && !! editedPost ) );
+	const isReady = postWithTemplate
+		? !! contextPost && !! editedPost
+		: !! editedPost;
```

Comment:
I just rewrote the condition here for readability. I'm not sure why this condition isn't enough? What other stuff are we waiting for? It's not clear. If there's more, can't we be explicit about it?